homeassistant/components/media_player/__init__.py

```diff
@@ -937,12 +946,15 @@ def get(self, request, entity_id):
                          request.GET.get('token') == player.access_token)
 
         if not authenticated:
-            return web.Response(status=401)
+            return web.Response(status=401),
```

Comment:
What's with this comma? That is introducing a bug. You're now returning a tuple but we explicitly test the returned response against being an instance of `web.Response`